scripts/_lines_of_code.py

```diff
@@ -55,58 +55,46 @@ class ContributedRepository:
     def __init__(self, contributed_to: Repository, user):
         self.total_lines_of_addition_in_contribution = 0
         self.total_lines_of_deletion_in_contribution = 0
-        self.excepted_lines_of_addition_in_contribution = 0
-        self.excepted_lines_of_deletion_in_contribution = 0
         self.name = contributed_to.full_name
 
-        for contribution in contributed_to.get_stats_contributors():
-            if contribution.author.id == user.id:
-                for week in contribution.weeks:
-                    self.total_lines_of_addition_in_contribution += week.a
-                    self.total_lines_of_deletion_in_contribution += week.d
-        if not any([self.total_lines_of_addition_in_contribution, self.total_lines_of_deletion_in_contribution]):
-            raise NotContributedError('User have not contributed to this repository.')
-        
+        if (contributed_to.get_commits().totalCount >= 100):
```

Comment:
I think brackets are not required here...
```suggestion
        if contributed_to.get_commits().totalCount >= 100:
```